frontend/packages/fusion-frontend/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "rootDir": "src",
-    "outDir": "."
+    "outDir": ".",
+    "skipLibCheck": true
```

Comment:
Is it necessary? Does the absence of this rule break something?

Reply:
Without this in `fusion-frontend` package the type check (`npm run typecheck`) fails with:
```
node_modules/fetch-mock/esm/client.d.ts:694:16 - error TS2649: Cannot augment module 'fetchMock' with value exports because it resolves to a non-module entity.

694 declare module 'fetch-mock/esm/client' {
                   ~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```
And this looked like something that can't be fixed in our code so I added this to make it pass.

I added the same option to the form package for consistency even though it didn't have similar errors for now.

Reply:
Ah, I see, thanks. Looks like I was satisfied with the Intellij check and haven't tested the `typecheck` command properly.